docs/guidelines-declaring.md

```diff
@@ -0,0 +1,197 @@
+
+## Usual noise
+
+Generally speaking, noises are unwanted content in versions. At the moment we found three types of noise.
+
+### Irrelevant content
+
+The first type of noise we try to remove is content that is not **relevant legally speaking**, and that harms **document readability**.
+
+**CSS selectors** are a first step as they permit to select an area instead of the whole page, but they let pass through content such as headers, footers, buttons, drop-down lists...
+
+**Filtering** permits to get rid of the remaining irrelevant content.
+
+A drop-down list let user select which document he would like to see but this list doesn't interest us in the final document.
+
+**HTML file :**
+```html
+<div class="filter-holder">
+  <select class="filter-options">
+      <option value="https://www.redditinc.com/policies/user-agreement" selected>User Agreement</option>
+      <option value="https://www.redditinc.com/policies/privacy-policy">Privacy Policy</option>
+      <option value="https://www.redditinc.com/policies/content-policy">Content Policy</option>
+      <option value="https://www.redditinc.com/policies/broadcasting-content-policy">Broadcasting Content Policy</option>
+  </select>
+</div>
+<h1>Reddit User Agreement</h1>
+```
+**Markdown file :**
+```
+User Agreement Privacy Policy Content Policy Broadcasting Content Policy Moderator Guidelines Transparency Report 2017 Transparency Report 2018 Guidelines for Law Enforcement Transparency Report 2019
+
+Reddit User Agreement
+=====================
+```
+**Wished Markdown file :**
+```
+
+Reddit User Agreement
+=====================
+```
+**Filter in Javascript :**
+```javascript
+export function removeOptionsList(document) {
+  document.querySelectorAll('.filter-holder').forEach(element => element.remove());
+}
+```
+
+### Invisible HTML elements
+
+The second type of noise we try to remove are **elements invisible in the original HTML page** that **become visible in Markdown** or that **disrupt Markdown rendering**.
+
+- **Invisible elements that become visible in Markdown** are usually hidden in the original page via CSS stylesheets.
+
+An invisible paragraph (with display:none style) visible in the Markdown.
+
+**HTML file :**
+```html
+<h1>Twitter Terms of Service</h1>
+<p style="display: none;">goglobalwithtwitterbanner</p>
+```
+**Markdown file :**
+```
+Twitter Terms of Service
+========================
+goglobalwithtwitterbanner
+```
+**Wished Markdown file :**
+```
+Twitter Terms of Service
+========================
+```
+**Filter in Javascript :**
+```javascript
+export function removeNotDisplayedElements(document) {
+  document.querySelectorAll('[style="display: none;"]').forEach(element => element.remove());
```

Comment:
The examples of filters do not seem quite right to me when they seems to me that they could easily be replaced by a `remove`,  which is preferable.

Reply:
Thanks for this catch! I've rewritten these guidelines to direct the reader to the declaration documentation.

I wonder however about removing entirely this “usual noise” section. On the one hand, I like the idea that it takes the angle of “here is noise, what can I do about it” instead of a linear documentation, on the other hand I don't find it very well written and it is outdated. WDYT @Ndpnt @clementbiron?

Reply:
We should also decide what we do of the related [wiki page](https://github.com/ambanum/OpenTermsArchive/wiki/Usual-noise).

Reply:
The implementation in https://github.com/ambanum/OpenTermsArchive/pull/971/commits/9be0b0d1ff28226a76d33133232b1ad57de0e560 is good enough. The rest is out of scope of this PR.